src/js/model/primitives/Guide.js

```diff
@@ -1,9 +1,18 @@
+
 var Primitive = require('./Primitive'),
     model  = require('../'),
     lookup = model.primitive;
 
 var GTYPES = {AXIS: 1, LEGEND: 2},
     ORIENT = {x: 'bottom', y: 'left'};
+/**
+* Guide
+* @constructor
+* @param {string} gtype
+* @param {string} type
+* @param {string} scale
+*
+*/
```

Comment:
Do we care about having a preceding space?
```js
/**
 * Something
 */
```
vs
```js
/**
* something
*/
```
I am not sure whether this is enforceable via eslint